src/screens/Customers/CustomerProfile.tsx

```diff
@@ -133,35 +118,198 @@ function CustomerProfile(props: CustomerProps) {
           }}
           size="small"
         >
-          <AddIcon style={{ color: 'white' }} />
+          <FlashOnIcon style={{ color: 'white' }} />
         </IconButton>
       </div>
     );
   };
 
+  const getPaymentInfo = () => {
+    if (customer.meterType === "Inactive") {
+      balanceInfo = [{ number: UNDEFINED_AMOUNT, label: 'Remaining Balance', unit: '' }];
+      return (
+        <OutlinedCardList info={balanceInfo} primary={false} grayText={true} />
+      );
+    } else {
+      return (
+        <OutlinedCardList info={balanceInfo} primary={true} rightIcon={getPaymentButton()} />
+      );
+    }
+  }
+
+  const getReadingInfo = () => {
+    if (customer.meterType === "Analog Meter") {
+      return (
+        <div>
+          <OutlinedCardList info={readingInfo} primary={false} rightIcon={getAddButton()} />
+          <div className={classes.section}>
+            { /* Top Left */ }
+            <MeterInfoContainer
+              floatRight={false}
+              info={meterInfo[0]}
+              editable={true}
+            />
+            { /* Top Right */ }
+            <MeterInfoContainer
+              floatRight={true}
+              info={meterInfo[2]}
+            />
+            { /* Bottom Right */ }
+            <MeterInfoContainer
+              floatRight={true}
+              info={meterInfo[3]}
+            />
+            { /* Bottom Left */ }
+            <MeterInfoContainer
+              floatRight={false}
+              info={meterInfo[1]}
+            />
```

Comment:
Can the `if` statements just determine what to pass to the `MeterInfoContainer`s and just return 1 set of `MeterInfoContainers` instead of having 4 returns? I was thinking something along the lines of 

```
let meterInfo;
let readingInfo;
let topLeftGray;
let topRightGray;
let bottomRightGray;
let bottomLeftGray;

if (customer.meterType === 'Analog Meter') {
    meterInfo = .... 
    readingInfo = ...
    ...
} else if (customer.meterType === 'SmartMeter) {
    meterInfo = ...
    readingInfo = ...
    ...
} else {
    ...
}

return (
        <div>
          <OutlinedCardList info={readingInfo} primary={false} grayBackground={true} />
          <div className={classes.section}>
            { /* Top Left */ }
            <MeterInfoContainer
              floatRight={false}
              info={meterInfo[0]}
              gray={topLeftGray}
            />
            { /* Top Right */ }
            <MeterInfoContainer
              floatRight={true}
              info={meterInfo[2]}
              gray={topRightGray}
            />
            { /* Bottom Right */ }
            <MeterInfoContainer
              floatRight={true}
              info={meterInfo[3]}
              gray={bottomRightGray}
            />
            { /* Bottom Left */ }
            <MeterInfoContainer
              floatRight={false}
              info={meterInfo[1]}
              gray={bottomLeftGray}
            />
          </div>
        </div>
)

```

Cuts the amount of code by a lot and is easier to read and easier to maintain since there's only 1 return